src/osd/PrimaryLogPG.h

```diff
@@ -829,7 +829,8 @@ class PrimaryLogPG : public PG, public PGBackend::Listener {
     if (!to_req.empty()) {
       // requeue at front of scrub blocking queue if we are blocked by scrub
       for (auto &&p: to_req) {
-	if (write_blocked_by_scrub(p.first.get_head())) {
+	if (write_blocked_by_scrub(
+              hobject_t::turn_into_head(std::move(p.first->obs.oi.soid)))) {
```

Comment:
This looksl ike it's destroying the soid value in the obc!  That won't work...

Reply:
Thanks for pointing this out! `turn_into_head` was a thing from the era of `hobject_t`-storing `to_req`. :-/